### DISCUSSION TITLE
[Auth] Apple watch support

### DISCUSSION THREAD

**Discussion by vsvito420** [ente-io/ente#2771] [Enhancements]
I want to use my apple watch pls :) also dont forget a copy button maybe the copy gets transfered to iphone

**Comment by SaijoGeorge** [comment]
That is what is holding me back from switching

**Reply by nordentesla** [reply]
Likewise!

**Comment by mallomar** [comment]
Would also really love Apple Watch support.

**Comment by escapewindow** [comment]
Dup of #485 ?

**Reply by mallomar** [reply]
WearOS is Android's watch OS. watchOS is Apple's.

**Comment by pizzapi2012** [comment]
Please add support.

**Comment by nordentesla** [comment]
Hello all! To all the devs participating in developing this growing ecosystem, thank you very much for all of your hard work, I come from Authy and would like to migrate to this MFA system, the only thing that is holding me back is the watchOS support (Apple Watch) for quick MFA viewing especially for multiple accounts, thank you.
